src/components/Button/Button.js

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+function nextPage(pageLink) {
```

Comment:
the function here is called `nextPage` but saved in `Button.js`, this is a bit confusing. You should adjust one of these names to make the default export match the filename. 
more info: https://javascript.info/import-export#export-default
especially the second paragraph: https://javascript.info/import-export#a-word-against-default-exports


`pageLink` is also a bit confusing since it is used as if it was a `props` object, but declared as a parameter, I recommend checking on that and either rename or destructure the property. It looks like `pageLink` isn't doing anything here, try to remove it and see if the component still works

Reply:
this is not incorrect,` props` can be named anything. It's just difficult to understand for someone else and the name `pageLink` doesn't make any sense here :)